library/board.py

```diff
@@ -147,12 +159,81 @@ def redraw_gridlines(
             else:
                 print(term.move_x(x) + crosses)
 
+    def victory_subgrid(
+        self,
+        term: blessed.Terminal,
+        subgrid: npt.NDArray[np.str_],
+        start_coords: typing.Tuple[int, int],
+        winner: str,
+    ) -> None:
+        """Takes the subgrid and draws a victory symbol over that grid"""
+        x, y = start_coords
+        if winner == "None":
+            return None
+        if winner == "X":
+            print(
+                term.move_xy(x + 1, y)
+                + f"{term.on_color_rgb(0, 0, 200)}{subgrid[0, 0]} {term.normal}"
+                + term.move_x(x + 9)
+                + f"{term.on_color_rgb(0, 0, 200)}{subgrid[0, 2]} {term.normal}"
+            )
+            print(
+                term.move_x(x + 3)
+                + f"{term.on_color_rgb(0, 0, 200)}┼─{term.normal}"
+                + term.move_x(x + 7)
+                + f"{term.on_color_rgb(0, 0, 200)}┼─{term.normal}"
+            )
+            print(
+                term.move_x(x + 5)
+                + f"{term.on_color_rgb(0, 0, 200)}{subgrid[1, 1]} {term.normal}"
+            )
+            print(
+                term.move_x(x + 3)
+                + f"{term.on_color_rgb(0, 0, 200)}┼─{term.normal}"
+                + term.move_x(x + 7)
+                + f"{term.on_color_rgb(0, 0, 200)}┼─{term.normal}"
+            )
+            print(
+                term.move_x(x + 1)
+                + f"{term.on_color_rgb(0, 0, 200)}{subgrid[2, 0]} {term.normal}"
+                + term.move_x(x + 9)
+                + f"{term.on_color_rgb(0, 0, 200)}{subgrid[2, 2]} {term.normal}"
+            )
+        if winner == "O":
+            print(
+                term.move_xy(x + 3, y)
+                + f"{term.on_color_rgb(254, 0, 0)}│ {subgrid[0, 1]} │{term.normal}"
+            )
+            print(
+                term.move_x(x + 2)
+                + f"{term.on_color_rgb(254, 0, 0)}─{term.normal}"
+                + term.move_x(x + 8)
+                + f"{term.on_color_rgb(254, 0, 0)}─{term.normal}"
+            )
+            print(
+                term.move_x(x + 2)
+                + f"{term.on_color_rgb(254, 0, 0)} {term.normal}"
+                + term.move_x(x + 8)
+                + f"{term.on_color_rgb(254, 0, 0)} {term.normal}"
+            )
+            print(
+                term.move_x(x + 2)
+                + f"{term.on_color_rgb(254, 0, 0)}─{term.normal}"
+                + term.move_x(x + 8)
+                + f"{term.on_color_rgb(254, 0, 0)}─{term.normal}"
+            )
+            print(
+                term.move_x(x + 3)
+                + f"{term.on_color_rgb(254, 0, 0)}│ {subgrid[0, 1]} │{term.normal}"
+            )
+
     def redraw_subgrid(
         self,
         term: blessed.Terminal,
         subgrid: npt.NDArray[np.str_],
         number: str,
         color: str,
+        winner: str,
```

Comment:
See above comment about stringly typed things.

Reply:
See reply to comment above